src/components/Pages/Trending/Trending.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import CustomPagination from "../../Pagination/CustomPagination";
+import SingleContent from "../../SingleContent/SingleContent";
+import "./Trending.css";
+
+const Trending = () => {
+  const [page, setPage] = useState(1);
+  const [content, setContent] = useState([]);
```

Comment:
I'd change `content` to have a more meaningful name. Maybe `trendingMovies`?

Reply:
I changed the topic and the button name to new releases.